hardware_interface/include/hardware_interface/system_interface.hpp

```diff
@@ -18,7 +18,7 @@
 #include <memory>
 #include <string>
 #include <vector>
-
+#include <iostream>
```

Comment:
debug leftover?

Reply:
Yup, thought I had gotten all of them removed, my bad.